google_cse.module

```diff
@@ -218,7 +218,9 @@ function google_cse_sitesearch_default() {
  * Adds custom submit handler for search form.
  */
 function google_cse_form_search_form_alter(&$form, &$form_state, $form_id) {
-  google_cse_sitesearch_form($form);
+  if ($form_id == 'search_form') {
```

Comment:
Use this http://cgit.drupalcode.org/drupal/tree/core/modules/search/src/Form/SearchBlockForm.php#n77
to check what page is target in the block